crates/matrix-sdk-base/src/latest_event.rs

```diff
@@ -65,6 +65,19 @@ impl LatestEventValue {
         }
     }
 
+    /// Check whether an event is remote-like, i.e. has been sent to a
+    /// homeserver (by this client, or another), i.e. is **not**
+    /// [`LocalIsSending`] nor [`LocalCannotBeSent`].
+    ///
+    /// [`LocalIsSending`]: LatestEventValue::LocalIsSending
+    /// [`LocalCannotBeSent`]: LatestEventValue::LocalCannotBeSent
+    pub fn is_remote_like(&self) -> bool {
+        match self {
+            Self::LocalIsSending(_) | Self::LocalCannotBeSent(_) => false,
+            Self::LocalHasBeenSent(_) | Self::Remote(_) | Self::None => true,
+        }
+    }
```

Comment:
Perhaps a better name would be `has_been_sent()` though not sure how `LatestEventValue::None` fits into this.

Reply:
Changing for `is_unsent` as agreed together in the Matrix room.